src/subdomains/core/buy-crypto/process/services/buy-crypto-preparation.service.ts

```diff
@@ -139,7 +144,7 @@ export class BuyCryptoPreparationService {
 
         if (entity.cryptoInput) await this.payInService.updateAmlCheck(entity.cryptoInput.id, entity.amlCheck);
 
-        await this.buyCryptoWebhookService.triggerWebhook(entity);
+        if (amlCheckBefore !== entity.amlCheck) await this.buyCryptoWebhookService.triggerWebhook(entity);
 
```

Comment:
Do we need the same for buy-fiat?

Reply:
Hehe thought the same 🤔 
Don't know, is there no trigger for webhooks in buyFiat amlCheck or is it only me who doesn't see it atm